connect-mixin.js

```diff
@@ -29,15 +30,37 @@ export const connect = (store) => (baseElement) => class extends baseElement {
       }
   }
 
+    /**
+   * Setter of active property.
+   */
+  set active(val) {
```

Comment:
This seems backward-incompatible change. We should set default value of `active=true` from the constructor. If any integrator wants to set it's initial value to `false` then can do it from it's own constructor.

Reply:
Yes, that's why I have checked with  `active "===" false` condition. If active property is not defined, then it will not prevent stateChanged call.